test/team_collection_test.rb

```diff
@@ -0,0 +1,22 @@
+require './test/test_helper'
+require './lib/team'
```

Comment:
Test if line can be removed, we only call on TeamCollection within this test file.

Reply:
Line 2 is also referencing /lib/team and the actual file is lib/teams as noted above.